app/models/status_crowdsource.rb

```diff
@@ -40,7 +40,8 @@ class StatusCrowdsource < Status
 
   def calculate_status
     votes = total_votes
-    return unless votes.any?
+    arr = votes.to_a
```

Comment:
Why?
The previous method would only perform a simple SELECT with a COUNT (which is fast) and this one not only gets all the data, it also create an array of objects with it.

Reply:
but you later use all the votes anyway if there are any, so you do two queries, no?

Reply:
this way you do the query once and then just handle things in ruby. that's what I understood :D and the log in the console became shorter.

Reply:
It depends on how many stores have new entries from users. If they're the majority, we save one query in the majority, if they're not, we spend more time (cause for the stores that have no entries, it will make only a single quick query, instead of a big query and creating all the model objects).

Reply:
what if we do a pop or access the first element? if it's empty we do a query, if it's not empty we do a query and we already have the objects ready to use? but happy to leave it out from the PR: can I merge the rest?